src/toTransform.js

```diff
@@ -94,32 +96,33 @@
 
   function convertPathString (properties) {
     var offsetPath = internalScope.offsetPathParse(properties['offsetPath']);
-    var closedLoop = isClosedLoop(offsetPath);
 
     var offsetDistance = internalScope.offsetDistanceParse(properties['offsetDistance']);
     if (offsetDistance === undefined) {
       offsetDistance = {value: 0, unit: 'px'};
     }
 
     pathElement.setAttribute('d', offsetPath.path);
-    var totalPathLength = pathElement.getTotalLength();
 
-    var currentOffsetDistance = getPathStringOffsetDistance(offsetPath, pathElement, offsetDistance, 0);
+    var currentOffsetDistance = getPathStringOffsetDistance(offsetPath, pathElement, offsetDistance);
+    var currentPoint = pathElement.getPointAtLength(currentOffsetDistance);
 
     var epsilon = 0.0001;
```

Comment:
perhaps a comment explaining this magic number? E.g. 'arbitrary small number to find the angle at this point in the path'